_develop/connect/overview.md

```diff
@@ -8,22 +8,48 @@ permalink: /:collection/:path.html
 * TOC
 {:toc}
 
-Blockstack Connect is a JavaScript library for integrating your application with Stacks v2. With Connect, you get some big benefits:
+Blockstack Connect is a JavaScript library for integrating Blockstack authentication and smart contracts into your app.
 
-- Pre-built UI to introduce your users to Blockstack, before they authenticate
-- Integration with our new authenticator
-- Ability to interact with the Stacks 2.0 blockchain, with support for smart contracts and other transactions.
+The library empowers you to:
 
-## Demo
+- Register new users with a pre-built onboarding flow that quickly educates them as to the privacy benefits of using your app with Blockstack and provisions a "Secret Key" that secures their identity and data against the Stacks blockchain.
+- Authenticate users when they return to your app using that same Secret Key.
+- Prompt users to sign transactions with smart contracts as written in Clarity and published to the Stacks blockchain.
 
-To get a test feel for the user experience of using Connect, you can use [Banter](https://banter.pub).
+## See it in action
 
-## How does this compare to `blockstack.js`?
+Here's what it looks like to register new users into an app with Blockstack Connect:
+
+**insert video**
+
+As you can see, you simply need to add a "Get started" option to your app homepage and the library will display an introductory modal in the context of your app that explains the privacy benefits of using Blockstack and prepares users to receive a Secret Key instead of entering a traditional email and password.
```

Comment:
this is a mouthful and I don't understand what a get started option in my app homepage would be

Reply:
Maybe this is where a short video/gif would tell the story much better.

Reply:
The video should help once we've embedded it above. I'll work on the language here to make it clearer as well.